### ISSUE TITLE
!todo Dragon tribal inflamed don't work with Grady.

### ISSUE THREAD

**Issue by karlwilcox** [drawshield/Drawshield-Code#645]
!todo Dragon tribal inflamed don't work with Grady.

**Comment by karlwilcox** [comment]
Depends a bit you mean by "doesn't work"(!)

Firstly, "inflamed" is a *feature*, which in DrawShield terms is a (usually quite small) part of a charge that can take a different colour. Because it is (usually) such a small part of a charge (and because coding it any other way is quite hard) features can only be plain colours (e.g. it would be pretty pointless to do something like a "lion gules armed ermine", the actual ermine spots would be really tiny...). 'inflamed' is a feature, so it only takes a plain colour, not grady. (Also remember that "grady" has absolutely NO heraldic precedent, i just added it because I thought it looked nice...)

There **is** a short-coming in the parser in that it **should** recognise more complex tinctures for features but it should then ignore them and produce a proper error message.

Now, I will admit that in the particular case of the tribal dragon the "inflamed" feature is actually bigger than the rest of the dragon so my earlier argument probably isn't valid but it is still hard to code up. Making the whole of the dragon grady looks nice though...

In short, I will try to fix the parser problem noted above, but the original issue is unlikely to be fixed, sorry.
